src/js/components/Select/doc.js

```diff
@@ -161,3 +161,43 @@ export const doc = Select => {
 
   return DocumentedSelect;
 };
+
+export const themeDoc = {
+  'select.background': {
+    description: 'The background color used for Select.',
+    type: 'string',
+    defaultValue: undefined,
+  },
+  'select.container.extend': {
+    description:
+      'Any additional style for the container of the Select component.',
+    type: 'string | (props) => {}',
+    defaultValue: undefined,
+  },
+  'select.control.extend': {
+    description:
+      'Any additional style for the control of the Select component.',
+    type: 'string | (props) => {}',
+    defaultValue: undefined,
+  },
+  'select.icons.color': {
+    description: 'The color used for Select icons.',
+    type: 'string | { dark: string, light: string }',
+    defaultValue: undefined,
+  },
+  'select.icons.down': {
+    description: 'The down icon to use for opening the Select.',
+    type: 'React.element',
+    defaultValue: '<FormDown />',
+  },
+  'select.searchInput': {
+    description: `The different properties of the Select input field.`,
```

Comment:
If you look at the way this is used at line 312 of SelectContainer.js, it isn't an object with keys that can be used as properties, it is a component. For example, take a look at how select/stories/theme.js sets it to be "SearchInput", which is a component defined in select/stories/components/SearchInput.js.